examples/park_app.py

```diff
@@ -33,10 +33,9 @@ class Point:
     def from_dict(cls, data: Dict) -> "Point":
         if "lat" in data:
             return cls(float(data["lat"]), float(data["lng"]))
-        elif "latitude" in data:
+        if "latitude" in data:
```

Comment:
Are we expecting that these would both happen to switch it from being an `elif`?

Reply:
No, just because 
```
if foo: return
elif bar: return
```
is equivalent to:
```
if foo: return
if bar: return
```

https://docs.astral.sh/ruff/rules/superfluous-else-return/